docs/user_guide/tunnel_server.md

```diff
@@ -0,0 +1,208 @@
+# Tunnel Server
+
+## Introduction
+
+`gNMIc` supports gNMI Dial-out as defined by [`openconfig/grpctunnel`](https://github.com/openconfig/grpctunnel).
+
+`gNMIc` embeds a tunnel server to which the gNMI targets register. Once registered, `gNMIc` triggers the request gNMI RPC towards the target via the established tunnel.
+
+This use case is described [here](https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmignoissh-dialout-grpctunnel.md#gnmi-collector-with-embedded-tunnel-server)
+
+## Server operation
+
+When running a Subscribe RPC using `gNMIc` with the flag `--use-tunnel-server`,`gNMIc` starts by running the Tunnel server as defined under `tunnel-server`.
```

Comment:
maybe to clarify what tunnel-server place we are talking about here?

under `tunnel-server` **config section**?

Reply:
That's what `defined under` means, or is `configured under` better?